ui/component/fileActions/index.js

```diff
@@ -7,6 +7,7 @@ import {
   selectClaimIsNsfwForUri,
```

Comment:
:small_blue_diamond: (Minor polishing) Handle playlist -- currently can be added to queue and results in a video error without explanation.